setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = versations
+version = 0.1.1
+description = A package for syncing, logging, and sending messages to Matrix rooms.
+long_description = file:README.md
+long_description_content_type = text/markdown
+
+[options]
+packages = find:
+python_requires = >= 3.8
+package_dir =
+      = src
+install_requires =
+      click
+      colorama
+      matrix-nio[e2e]
+      pyyaml
+      rich
```

Comment:
Why is `rich` listed here?  There are no references to it in the code.

Reply:
i used it in development, nice catch.